server/src/main/java/com/genymobile/scrcpy/Server.java

```diff
@@ -251,6 +251,7 @@ public void uncaughtException(Thread t, Throwable e) {
         Options options = createOptions(args);
 
         Ln.initLogLevel(options.getLogLevel());
+        Ln.v("Using verbose log level");
```

Comment:
Should I drop this log and leave the `v()` method unused?

Reply:
I think you should drop it.